### PR TITLE
Add ACCESSORS = 1.0-no-warn

### PR DESCRIPTION
**Pull request by jamesarosen**
@shajith @jish

### CODE REVIEW

**Comment by shajith** [comment]
:+1:

**Comment by jish** [comment]
docs? https://github.com/zendesk/ember.js/blob/master/doc/accessors.md

File: packages/ember-handlebars/tests/backports/getPath_test.js [hunk 1]
```diff
@@ -21,14 +21,21 @@ module("Backported Ember.Handlebars.getPath", {
...
+  Ember.ENV.ACCESSORS = "1.0-no-warn";
+  var o = { foo: { 'bar': 'baz' } };
   Ember.Handlebars.getPath(o, 'foo.bar');
   equal(warnings.length, 0);
 });
 
 test("warns on usage in 1.0 mode", function() {
   Ember.ENV.ACCESSORS = "1.0";
-  var o = { 'foo.bar': 'baz' };
+  var o = { foo: { 'bar': 'baz' } };
```
Comment (jish) [inline packages/ember-handlebars/tests/backports/getPath_test.js:38]: I don't understand this change. Was the test written wrong before? Did the behavior change?

**Comment by jamesarosen** [comment]
@jish I'm _sure_ I wrote docs for this. I don't see the changes here though.

File: packages/ember-handlebars/tests/backports/getPath_test.js (hunk 1 above)
Comment (jamesarosen) [inline packages/ember-handlebars/tests/backports/getPath_test.js:38]: My reading of the test indicates it was wrong before.

File: doc/accessors.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ allowed values:
  * `null` (the default) -- `get`, `set`, `getPath`, and `setPath` work per 0.9.8.1
  * `"0.9-dotted-properties"` -- `get` and `set` will warn if called with a property name
    containing a `.`
+ * `"1.0-no-warn"` -- `get` and `set` work with nested properties; `getPath`
+   and `setPath` do not emit deprecation warnings
```
Comment (jamesarosen) [inline doc/accessors.md:11]: @jish yeah, I knew the docs were here.

Comment (jish) [inline doc/accessors.md:11]: Oh, how did I miss that? =o

Carry on... ;)

**Comment by jamesarosen** [comment]
@jish and I found some bugs here. I'm going to rework this.

**Comment by jamesarosen** [comment]
Closing in favor of #19

**Pull request closed without merging**